packages/block-editor/src/store/reducer.js

```diff
@@ -1650,6 +1650,15 @@ export function highlightedBlock( state, action ) {
 	return state;
 }
 
+export function hoveredBlocks( state = [], action ) {
+	switch ( action.type ) {
+		case 'SET_HOVERED_BLOCKS':
+			return action.hoveredBlockIds;
```

Comment:
I wonder if we should make this an array of the actual block objects as opposed to just the clientIds.  And if so I wonder if we should do that here or in the selector. 🤔   I don't think it makes a big difference for our needs, but maybe this would be more useful if it contained more information so people wouldn't need to use `getBlock` or `getBlockName` to analyze it. 🤷‍♀️

Reply:
i think the clientIds make sense because when block change, these might be stale.

Reply:
True, but that wouldn't be stale if we retrieved the blocks from the ids when the selector runs?  either way its probably not necessary.